src/lib/definition-parser.ts

```diff
@@ -69,27 +107,44 @@ function getTypesVersionsAndPackageJson(ls: ReadonlyArray<string>): LsMinusTypes
     return { remainingLs, typesVersions, hasPackageJson: withoutPackageJson.length !== ls.length };
 }
 
-export function parseMajorVersionFromDirectoryName(directoryName: string): number | undefined {
-    const match = /^v(\d+)$/.exec(directoryName);
-    // tslint:disable-next-line no-null-keyword
-    return match === null ? undefined : Number(match[1]);
+/**
+ * Parses a directory name into a version that either holds a single major version or a major and minor version.
+ *
+ * @example
+ *
+ * ```ts
+ * parseVersionFromDirectoryName("v1") // { major: 1 }
+ * parseVersionFromDirectoryName("v0.61") // { major: 0, minor: 61 }
+ * ```
+ */
+export function parseVersionFromDirectoryName(directoryName: string): TypingVersion | undefined {
+    const match = /^v(\d+)(\.(\d+))?$/.exec(directoryName);
+    if (match === null) {
+        return undefined;
+    } else {
+        return {
+            major: Number(match[1]),
+            minor: match[3] !== undefined ? Number(match[3]) : undefined, // tslint:disable-line strict-type-predicates (false positive)
+        };
+    }
 }
+
 function combineDataForAllTypesVersions(
     typingsPackageName: string,
     ls: ReadonlyArray<string>,
     fs: FS,
-    oldMajorVersion: number | undefined,
-): TypingsDataRaw {
+    directoryVersion: TypingVersion | undefined,
+): Omit<TypingsDataRaw, "libraryVersionDirectoryName"> {
```

Comment:
why do we need to omit libraryVersionDirectoryName here?

Reply:
`libraryVersionDirectoryName` is not determined in this function. I could have passed the value in, only to include it in the object and return it, but that felt like bolting it on. I can go either way, if you have a preference.